### ISSUE TITLE
Pit/In-Car Menu lagging

### ISSUE THREAD

**Issue by andreasdahl1987** [andreasdahl1987/DahlDesignProperties#57]
Noticed the Pit Menu and In-Car menu now have about 0.5 second delay when changing position in the menu. The delay is uniform; the same delay every time. Tried on a different computer, much less delay here - but still a little bit, maybe 0.2 seconds. Other dashboard actions such as launch screen respond instantly. 

Tried version 1.11, was working without any delay. 

@photon1503 Have you had the samme issue, got any idea what it might be?

**Comment by photon1503** [comment]
@andreasdahl1987 I can check this after office. Can you somehow assign the issue to me?

**Comment by andreasdahl1987** [comment]
Assigned you, what does this do? 😅 
I'm guessing the issue was introduced in 1.13.2 with PR #54, the extraction of the rotary actions to a seperate class. I haven't fully tested it, might also get some more time tonight.

**Comment by photon1503** [comment]
@andreasdahl1987 
Checked now, but can't find anything in the code. The action will update the variable instantly and it's assigned with a delegate. It will even update if no game is running. You can check by opening "available properties" in SimHub and filter to i.e. PitMenu or InCarMenu.

There is a short delay introduced by SimHub itself, since the Joystick buttons are polled, not triggered. 

Only for the press type "LongPress" there is a ~500ms delay within SimHub, all others should work in near realtime.

I also can't reproduce the behaviour. Tested with different press types (Default, ShortAndLong, During). Also key repeats are executed.

**Comment by andreasdahl1987** [comment]
@photon1503 
Didnt think of testing the various press types :D I'm pretty sure I didn't use the long press type though. Sim computer is slowly falling appart, I think I'll format it tonight and give this another go 👍

**Comment by andreasdahl1987** [comment]
> Grabbed version 1.13.0 and can't see any difference in the delays here.
> 
> Please also check if you deleted your old bindings in Simhub under "Controls and events". Since the keys were renamed, maybe they are here now twice?
> 
> ![image](https://user-images.githubusercontent.com/14548927/207899741-ff1a47a6-ad27-44e4-94e7-9e99f18c56c9.png)

Yeah I did check there, I had LOADS of bindings to non-existing actions. Cleared all, made fresh bindings. No difference.

**Comment by photon1503** [comment]
And it is faster on same PC with old version?

**Comment by andreasdahl1987** [comment]
Yes, super responsive. That's what's strange. But there might be some issues with some old setting or something. After trying 1.11 and got everything working (after binding new keys in 1.11), the plugin wouldnt start up at all in 1.12, 1.13 or build from master branch. It was late and I didnt get more time to work on it. I'll try deleting all DahlDesign files in the simhub directory and start fresh.

**Comment by photon1503** [comment]
Weird. Now also tried 1.11.0, but the response is the same. Also take care if you are using Debug or Release build on the simrig.

**Comment by andreasdahl1987** [comment]
I've tried with both. But again, I might have posted this issue prematurely. I'll do some more tests in the evening. Thanks for having a look :)

**Comment by andreasdahl1987** [comment]
@photon1503 So I deleted all files in Simhub folder, started over with build from main branch. Unable to start the plugin like yesterday. It turns out Dashboard.cs was crashing because when going back to an earlier version changed the value of some string settings, such as the default left/right qualy/practice/race screens. Now this caused a string conversion problem. I added a commit to make sure this cant happen. 

The issue with the laggy pit menu/ incar menu was also resolved, likely because of deleting all the files.
